content/sensu-go/5.18/guides/securing-sensu.md

```diff
@@ -202,85 +203,18 @@ trusted-ca-file: "/path/to/ca.pem"
-
-{{< highlight shell >}}
-echo '{"CN":"CA","key":{"algo":"rsa","size":2048}}' | cfssl gencert -initca - | cfssljson -bare ca -
-echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","server auth","client auth"]}}}' > ca-config.json
-{{< /highlight >}}
-
-Then, using that CA, generate certificates and keys for each peer (backend server) by specifying their Common Name (CN) and their hosts.
-A `*.pem`, `*.csr` and `*.pem` will be created for each backend.
-
-{{< highlight shell >}}
-export ADDRESS=10.0.0.1,backend-1
-export NAME=backend-1
-echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" -profile=peer - | cfssljson -bare $NAME
-
-export ADDRESS=10.0.0.2,backend-2
-export NAME=backend-2
-echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" -profile=peer - | cfssljson -bare $NAME
-
-export ADDRESS=10.0.0.3,backend-3
-export NAME=backend-3
-echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" -profile=peer - | cfssljson -bare $NAME
-{{< /highlight >}}
-
-You will also create generate a *client* certificate that can be used by clients to connect to the etcd client URL.
-This time, you don't need to specify an address, only a CN (here, `client`).
-The files `client-key.pem`, `client.csr`, and `client.pem` will be created:
-
-{{< highlight shell >}}
-export NAME=client
-echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="" -profile=client - | cfssljson -bare $NAME
-{{< /highlight >}}
-
-If you have a Sensu license, you can also generate a certificate for agent TLS authentication:
-
-{{< highlight shell >}}
-export NAME=agent-1
-echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="" -profile=client - | cfssljson -bare $NAME
-{{< /highlight >}}
-
-See [etcd's guide to generating self-signed certificates][6] for detailed instructions for securing etcd.
-
-When you're finished, the following files should be created (the `*.csr` files are not used in this guide):
-
-{{< highlight shell >}}
-agent-1-key.pem
-agent-1.csr
-agent-1.pem
-backend-1-key.pem
-backend-1.csr
-backend-1.pem
-backend-2-key.pem
-backend-2.csr
-backend-2.pem
-backend-3-key.pem
-backend-3.csr
-backend-3.pem
-ca-config.json
-ca-key.pem
-ca.csr
-ca.pem
-client-key.pem
-client.csr
-client.pem
-{{< /highlight >}}
+## Next step: Run a Sensu cluster
 
-## Next steps
+Well done!
+Your Sensu installation should now be secured with TLS.
```

Comment:
Need to make sure this secure Sensu guide actually does configure TLS by the end of the guide. We may want to make this clearer with more direction.

Reply:
In Install content, need to add callouts that say "If you're doing this for anything beyond POC or devel env, we recommend securing with TLS. See the Generate Certificates guide > Secure > Cluster" (this should set them on the path). Need an alternate next step for people who don't want to set up a cluster.